pkg/specgen/generate/storage_freebsd.go

```diff
@@ -0,0 +1,22 @@
+package generate
+
+import (
+	"context"
+
+	"github.com/containers/common/libimage"
+)
+
+func imageRunPath(ctx context.Context, img *libimage.Image) (string, error) {
```

Comment:
If this is only gonna be called on `freebsd` why not simple return `"/var/run", nil`

Reply:
FreeBSD hosts can run Linux images - in this case, we need to use ```"/run"``` since this is what the application is likely to expect.

Reply:
ah fair enough.